Formula/m/molten-vk.rb

```diff
@@ -179,9 +179,9 @@ def install
                         "MoltenVKShaderConverter"]
 
     inreplace "MoltenVK/icd/MoltenVK_icd.json",
-              "./libMoltenVK.dylib",
-              (lib/"libMoltenVK.dylib").relative_path_from(share/"vulkan/icd.d")
-    (share/"vulkan").install "MoltenVK/icd" => "icd.d"
+              "\"./libMoltenVK.dylib\"",
+              "\"#{lib}/libMoltenVK.dylib\""
```

Comment:
We could keep a relative path if preferred. Absolute path is simpler for inreplace. Relative path does avoid relocation in JSON file but wouldn't expect much impact unlike relocation in binary file.
```suggestion
              "./libMoltenVK.dylib",
              (lib/"libMoltenVK.dylib").relative_path_from(prefix/"etc/vulkan/icd.d")
```